packages/paste-website/src/pages/icon-system/index.mdx

```diff
@@ -0,0 +1,230 @@
+---
+title: Icon Guidelines
+description: Overview on how to use icons in the context of Paste projects.
+---
+
+import {graphql} from 'gatsby';
+import {Link} from 'gatsby';
+import {Box} from '@twilio-paste/box';
+import {Text} from '@twilio-paste/text';
+import {Anchor} from '@twilio-paste/anchor';
+import {Button} from '@twilio-paste/button';
+import {PlusIcon} from '@twilio-paste/icons/esm/PlusIcon';
+import {LoadingIcon} from '@twilio-paste/icons/esm/LoadingIcon';
+import {Callout, CalloutTitle, CalloutText} from '../../components/callout';
+import {DoDont, Do, Dont} from '../../components/DoDont';
+import Changelog from '@twilio-paste/icons/CHANGELOG.md';
+
+export const pageQuery = graphql`
+  {
+    mdx(fields: {slug: {eq: "/icon-system/"}}) {
+      headings {
+        depth
+        value
+      }
+    }
+  }
+`;
+
+<contentwrapper>
+
+<TableOfContents headings={props.data.mdx.headings} />
+
+<content>
+
+<h1>{props.pageContext.frontmatter.title}</h1>
+
+
+Icons are small graphical representation of a program or a function. They can be used to enhance the
+appearance of a web interface and break up the monotony of text. Use an icon only when it provides additional clarity or is necessary to draw attention to a UI element.
+If an icon is used to indicate hierarchy of a UI element, all elements at the same hierarchy should also be paired with an icon.
+
+
+<Callout>
+  <CalloutText>
+    You can view and play with icons on <Anchor href="https://twilio-labs.github.io/paste/?path=/story/overview-icons--list">our storybook icon list</Anchor>.
+    Our icon set is still limited, more will be adding over time. Please{' '}
+    <Anchor href="https://github.com/twilio-labs/paste/issues">file an icon request</Anchor> if you need an icon we
+    don't yet have. 
+  </CalloutText>
+</Callout>
+
+
+## Accessibility
+
+Using an icon as the only indicator of meaning is inaccessible. Different cultures may interpret the same image
+in different ways and visually impaired users may not see the icon. Here are some helpful guidelines to keep in mind:
+
+**Specify if an icon is decorative**
```

Comment:
Given this is under the accessibility section, we should probably use real headings here instead of fake ones. Keep the document hierarchy 2, so heading level 3s

Reply:
lol simon.